src/gs-updates-page.c

```diff
@@ -1627,6 +1628,8 @@ upgrade_trigger_finished_cb (GObject *source,
 		return;
 	}
 
+	gs_utils_update_install_timestamp ();
```

Comment:
Since this is not dependent on whether the user actually reboots the machine, maybe we should add this to the `gs_plugin_loader_process_thread_cb` too? (you can check if the function is `GS_PLUGIN_ACTION_UPGRADE_TRIGGER`)